### PR TITLE
add fhir-migrations to project

### PR DESCRIPTION
**Pull request by pbugni**
This should make it far easier to maintain scripts needed to affect existing and needed FHIR resources.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ click==8.0.1              # via flask
 flask-jwt-extended==3.24.1  # via patientsearch (setup.py)
 # pin to last hash of master, to remove requirement of freestanding JSON config file
 git+https://github.com/uwcirg/flask-oidc.git@ed4eb20#egg=flask-oidc
+git+https://github.com/uwcirg/fhir-migrations
```
Comment (ivan-c) [inline requirements.txt:7]: we might want to start tagging and pinning fhir-migrations, so we're prepared when we need to make breaking changes

```suggestion
git+https://github.com/uwcirg/fhir-migrations@0.1.0
```

I went ahead and [created a tag on fhir-migrations](https://github.com/uwcirg/fhir-migrations/releases/tag/0.1.0). I followed [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) format, but feel free to make another if you have another tag format in line

File: patientsearch/api.py [hunk 2]
```diff
@@ -30,7 +31,14 @@
...
 
-api_blueprint = Blueprint("patientsearch-api", __name__)
+api_blueprint = Blueprint("patientsearch-api", __name__, cli_group=None)
+
+
+@api_blueprint.cli.command("justtesting", help="yes, need help")
+@click.argument("name", required=True)
+def justtesting(name):
+    click.echo(f"hello {name}")
+    audit_entry("in justtesting()")
```
Comment (ivan-c) [inline patientsearch/api.py:41]: was this helpful in testing, or leftover?...

**Review by ivan-c — APPROVED** [review]
Looks great! Had a couple suggestions/comments but no blockers

File: patientsearch/api.py (hunk 2 above)
Comment (pbugni) [reply patientsearch/api.py:41]: 💯

**Pull request merged**